examples/resnet/cifar_input.py

```diff
@@ -55,8 +59,8 @@ def build_input(data, batch_size, train):
   labels_constant = tf.constant(data[1])
   image_size = 32
   depth = 3
-  num_classes = 10
-  image, label = tf.train.slice_input_producer([images_constant, labels_constant])
+  num_classes = 10 if dataset == 'cifar10' else 100
+  image, label = tf.train.slice_input_producer([images_constant, labels_constant], capacity=16 * batch_size)
```

Comment:
what is this number `16`? Can you either set that as a variable with a meaningful name or document what it is?